media/configuration-settings/main.js

```diff
@@ -526,12 +526,92 @@ const showOptions = function(event) {
     }
 }
 
-const exportValidation = function() {
+const pathValidator = function(target) {
+    for (let j=0;j<target.options.length;j++) {
+        if (target.options[j].optionName === 'input_path' && target.options[j].optionValue.trim() === '') {
+            vscode.postMessage({
+                command: 'alert',
+                text: `If you want to use ${target.type}, then input_path is required`
+            })
+            return true
+        }
+        if (target.options[j].optionName === 'output_path' && target.options[j].optionValue.trim() === '') {
+            vscode.postMessage({
+                command: 'alert',
+                text: `If you want to use ${target.type}, then output_path is required`
+            })
+            return true
+        }
+    }
+    return false
+}
 
+const baekendValidator = function(target) {
+    for (let j=0;j<target.options.length;j++) {
+        if (target.options[j].optionName === 'backend' && target.options[j].optionValue.trim() === '') {
+            vscode.postMessage({
+                command: 'alert',
+                text: `If you want to use ${target.type}, then backend is required`
+            })
+            return true
+        }
+    }
+    return false
 }
 
-const exportConfiguration = function() {
+const exportValidation = function() {
+    if (oneImport.use === true) {
+        let chosenModelIndex = -1
+        for (let i=0;i<oneImport.options.length;i++) {
+            if (oneImport.options[i].optionValue === true) {
+                chosenModelIndex = i
+                break
+            }
+        }
+        if (chosenModelIndex === -1 ) {
+            vscode.postMessage({
+                command: 'alert',
+                text: 'If you want to use oneImport, than you have to choose your framework'
+            })
+            return false
+        } else {
+            if (pathValidator(oneImportOptions[chosenModelIndex])) {
+                return false
+            }
+        }
+    }
+    if (optimize.use === true) {
+        if (pathValidator(optimize)) {
+            return false
+        }
+    }
```

Comment:
What about
`  if (optimize.use) return !pathValidator(optimize);`
I'm not sure it's possible with javascript

or you can just change value of return in pathValidator
true -> false
false -> true
Then change like this
`  if (optimize.use) return pathValidator(optimize);`